ReactiveXComponent/WebSocket/WebSocketSnapshotManager.cs

```diff
@@ -102,14 +105,6 @@ private void SendWebSocketSnapshotRequest(string stateMachine, string replyTopic
 
         private void SubscribeSnapshot(string replyTopic, out EventHandler<WebSocketMessageEventArgs> subscriptionHandler)
```

Comment:
How about renaming this method to something like _CreateSnapshotReplyHandler()_ and removing the line __webSocketClient.MessageReceived += subscriptionHandler;_ ?
Then each time you would do:
`CreateSnapshotReplyHandler(replyTopic, out subscriptionHandler);`
`_webSocketClient.MessageReceived += subscriptionHandler;`
`...`
`_webSocketClient.MessageReceived -= subscriptionHandler;`